README.md

```diff
@@ -1 +1,102 @@
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+{'valkrets': [], 'bild_url_192': [], 'uppgift': [], 'intressent_id_check': []}, 
+
+{'valkrets': ['Stockholms kommun'], 'bild_url_192': ['http://data.riksdagen.se/filarkiv/bilder/ledamot/979e9852-69ed-4077-8140-52ab99414a6c_192.jpg'], 'uppgift': [], 'intressent_id_check': ['014386615025']}, 
+
+{'valkrets': ['Stockholms kommun'], 'bild_url_192': ['http://data.riksdagen.se/filarkiv/bilder/ledamot/979e9852-69ed-4077-8140-52ab99414a6c_192.jpg'], 'uppgift': [], 'intressent_id_check': ['014386615025']}, 
+
+{'valkrets': ['Stockholms kommun'], 'bild_url_192': ['http://data.riksdagen.se/filarkiv/bilder/ledamot/979e9852-69ed-4077-8140-52ab99414a6c_192.jpg'], 'uppgift': [], 'intressent_id_check': ['014386615025']}, 
+
+{'valkrets': ['Stockholms kommun'], 'bild_url_192': ['http://data.riksdagen.se/filarkiv/bilder/ledamot/5b7ac02c-0819-426d-bc5d-41b8c21ca4dd_192.jpg'], 'uppgift': [], 'intressent_id_check': ['0216534495014']}
+]
+```
+
+## Approach
+>> Speech data already provides the list of latest ten speeches.
+
+>> Merger the relevant members data to the speeches data.
```

Comment:
What do you mean by `Merger`? Do you mean `merge`?

Reply:
change made